packages/loader/container-loader/src/protocol.ts

```diff
@@ -75,11 +75,12 @@ export class ProtocolHandler extends ProtocolOpHandler implements IProtocolHandl
 		message: ISequencedDocumentMessage,
 		local: boolean,
 	): IProcessMessageResult {
-		const client: ISequencedClient | undefined = this.quorum.getMember(message.clientId);
+		const client: ISequencedClient | undefined =
+			message.clientId === null ? undefined : this.quorum.getMember(message.clientId);
```

Comment:
Nit (could be a follow up PR) this is only used inside the `if`, it should move there and that simplifies typing a bit.